lib/DDG/Spice/Statista.pm

```diff
@@ -0,0 +1,26 @@
+package DDG::Spice::Statista;
+# ABSTRACT: Returns statistics from statista.com.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "statistics about soccer";
+description "Shows example statistics";
+name "Statista";
+category "facts";
+
+my $limit = 16;
+my $lang = 2;
+
+triggers startend => "statistic", "statistics", "stats";
```

Comment:
>Do you mean the first three reports (tiles) that a user clicks through will be free, but any more >would hit a paywall?

>Or do you mean literally the first 3 tiles are free links, and tiles 4+ are paid?

You won't see which content is free or paid for and it's not sorted in any way, but if a user is coming to statista.com from DuckDuckGo he will have access to upto 3 "paid for" contents for free.